sopel/modules/reddit.py

```diff
@@ -328,3 +384,38 @@ def get_channel_spoiler_free(bot, trigger):
         bot.say('%s is flagged as spoiler-free' % channel)
     else:
         bot.say('%s is flagged as spoilers-allowed' % channel)
+
+
+@rule(r'^(?P<prefix>r|u)\/(?P<id>[a-zA-Z-_]+)')
+def reddit_slash_info(bot, trigger):
+    searchtype = trigger.group('prefix')
+    match = trigger.group('id')
+    if searchtype == "r":
+        return subreddit_info(bot, trigger, match, is_command=True)
+    elif searchtype == "u":
+        return redditor_info(bot, trigger, match, is_command=True)
+
+
+@commands('subreddit')
+@example('.subreddit plex')
+def subreddit_command(bot, trigger):
+    # require input
+    if not trigger.group(2):
+        return bot.reply('You must provide a subreddit name.')
+
+    # subreddit names do not contain spaces
+    match = trigger.group(3)
+    return subreddit_info(bot, trigger, match, is_command=True)
+
+
+# If you change this, you'll have to change some other things...
```

Comment:
This comment is so far from its original purpose, we might as well just not keep it.

```suggestion
```